src/components/Inputs/input.module.scss

```diff
@@ -1438,6 +1441,109 @@
     &.input-stretch {
         width: 100%;
     }
+
+    &.is-expandable {
+        &.input-small .expandable-wrapper {
+            width: $input-small-size;
+        }
+
+        &.input-medium .expandable-wrapper {
+            width: $input-medium-size;
+        }
+
+        &.input-large .expandable-wrapper {
+            width: $input-large-size;
+        }
```

Comment:
Is there a way where we can do this that the implementation dictates the appropriate sizes? It seems too opinionated to hard code widths, even when giving three options.. and particular for the expanded state which will always be 500px below.